docker-compose.yml

```diff
@@ -64,21 +64,21 @@ services:
       - LOGIN_RADIUS_PRIVATE_KEY_PASS
       - LOGIN_RADIUS_KEYSTORE_PASS
       - LOGIN_RADIUS_KEYSTORE_NAME
-      - CIVIFORM_APPLICANT_IDP
-      - CIVIFORM_ADMIN_IDP
+      - CIVIFORM_APPLICANT_IDP=generic-oidc
       # Hard coded in test-support/test_oidc_provider.js
       - IDCS_CLIENT_ID=foo
       - IDCS_SECRET=bar
       - IDCS_DISCOVERY_URI=http://dev-oidc:3390/.well-known/openid-configuration
       - ADFS_CLIENT_ID
       - ADFS_SECRET
-      - APPLICANT_OIDC_PROVIDER_LOGOUT
+      - APPLICANT_OIDC_PROVIDER_LOGOUT=true
       - APPLICANT_OIDC_POST_LOGOUT_REDIRECT_PARAM
-      - APPLICANT_OIDC_LOGOUT_CLIENT_PARAM
+      - APPLICANT_OIDC_LOGOUT_CLIENT_PARAM=client_id
+      - APPLICANT_OIDC_OVERRIDE_LOGOUT_URL
       - APPLICANT_OIDC_PROVIDER_NAME
-      - APPLICANT_OIDC_CLIENT_ID
-      - APPLICANT_OIDC_CLIENT_SECRET
-      - APPLICANT_OIDC_DISCOVERY_URI
+      - APPLICANT_OIDC_CLIENT_ID=standard
+      - APPLICANT_OIDC_CLIENT_SECRET=bar
```

Comment:
Not blocking: I'd consider a different value here such as `fake-oidc-client-secret` or something along those lines. That way if something ends up in logs, it's easier to find the reference than searching for `bar`.

Reply:
Good idea!

Reply:
Done